src/common/s3_adapter.h

```diff
@@ -87,8 +87,20 @@ struct S3AdapterOption {
     uint64_t bpsWriteMB;
 };
 
-void InitS3AdaptorOption(Configuration *conf,
-    S3AdapterOption *s3Opt);
+struct FsS3Option {
+    // should get from mds
+    std::string ak;
+    std::string sk;
+    std::string s3Address;
+    std::string bucketName;
+    uint64_t blockSize;
+    uint64_t chunkSize;
+};
```

Comment:
you shouldn't put this option in here, I think s3 adapter is just put object to s3 or get object from s3, it shouldn't care about what is fs option.

Reply:
> you shouldn't put this option in here, I think s3 adapter is just put object to s3 or get object from s3, it shouldn't care about what is fs option.

this just for init, maybe the name is not well ,it should be like S3InfoOption?